style.css

```
@@ -459,3 +447,231 @@ textarea {
 hr {
   width: 100%;
 }
+
+.nav-elements {
+  display: none;
+}
+
+.desk-card-image {
+  display: none;
+}
+
+.facebook {
+  display: none;
+}
+
+.text-desk {
+  display: none;
+}
+
+.ruby {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .canopy {
+    display: none;
+  }
+
+  .text {
+    display: none;
+  }
+
+  body {
+    font-size: 20px;
+  }
+
+  .tallbar {
+    display: none;
+  }
+
+  .bars {
+    display: none;
+  }
+
```

- [ ] Kindly avoid code repetition in accordance to DRY principle, code repetition increases the size of the codebase unnecessarily, making it more complex and harder to manage. ℹ️ 

```suggestion
  .canopy,
  .text,
  .tallbar,
  .bars {
    display: none;
  }
```